scan_fmt_scxml/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "scan_scxml"
+version = "0.1.0"
+edition = "2024"
+
+[lib]
+name = "scan_scxml"  # The name of the target.
+crate-type = ["lib"]   # The crate types to generate.
+
+[dependencies]
+scan_core = { version = "0.1.0", path="../scan_core" }
+anyhow = { workspace = true }
+boa_ast = "0.20.0"
+boa_interner = "0.20.0"
+boa_parser = "0.20.0"
+log = { workspace = true }
+rand = { workspace = true }
+quick-xml = "0.37.5"
+thiserror = { workspace = true }
+logos = "0.15.0"
+chumsky = "0.10.1"
+csv = { workspace = true }
+flate2 = { workspace = true }
+scan_fmt_scxml = { path = "../scan_fmt_scxml" }
```

Comment:
A package cannot have itself as a dependency. You maybe wanted to import `scan_scxml`? Not sure it's needed anyway.